src/screens/SignTransactionSheet.tsx

```diff
@@ -946,7 +1003,9 @@ export const SignTransactionSheet = () => {
                         : i18n.t(i18n.l.walletconnect.simulation.buttons.confirm)
                     }
                     newShadows
-                    onPress={submitFn}
+                    onPress={performanceTracking
```

Comment:
in a similar vein to my comment above - could we put the `performanceTracking....executeFn` inside of the `submitFn` callback

Reply:
> it needs to wrap a function, so I'd have to create another function inside of the useCallback in order to wrap it? Not sure if that's preferred.